components/common/AppMenu.vue

```diff
@@ -46,6 +46,13 @@
       </nuxt-link>
     </div>
     <div>
+      <div class="app-menu-bug-report">
+        <a
+          href="https://github.com/tendermint/lunie-light/issues/new"
```

Comment:
The link would be for the Tendermint Lunie for all versions? Make this a config?